doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -3,351 +3,317 @@
+//approved
+.On {EAP}, the {PRODUCT} installer creates an incorrect password vault for the created datasource [https://issues.jboss.org/browse/RHPAM-2407[RHPAM-2407]]
+
+Issue: If you use the installer to install {PRODUCT} on {EAP} and you configure database settings, the `datasource` element in the `standalone` files is wrong and connection to the database fails.
+
+Steps to reproduce:
 
-Workaround: In the *Model* tab, enter the values as shown in the following example:
+. Use the installer to install {KIE_SERVER} only onto {EAP}.
+. When prompted, click *Configure Advanced Properties* -> *Configure Database Settings*.
+. Open the `standalone.xml` or `standalone-full.xml` file to verify user credentials.
 
+Expected result: The datasource element in the `standalone` files should look similar to the following, where `<NEWDATASOURCE>` is the name of your database:
 [source]
 ----
-((red), (blue))
+<password>${VAULT::datasource.<NEWDATASOURCE>::password::1}</password>
 ----
 
+Actual Result: The datasource element in the `standalone` files looks similar to the following:
 
+[source]
+----
+<password>:datasource.<NEWDATASOURCE>::password::1</password>
+----
 
-.A guided scorecard cannot be tested with the Test Scenario (Legacy) asset [https://issues.jboss.org/browse/RHPAM-2307[RHPAM-2307]]
+Workaround: Replace the `<password>` entry with the correct password vault entry for the database. In this example, `<NEWDATASOURCE>` is the name of your database:
 
-Issue: If a guided scorecard name begins with a lowercase letter, you cannot test the scorecard with the Test Scenario (Legacy) asset.
+[source]
+----
+${VAULT::datasource.<NEWDATASOURCE>::password::1}
+----
 
-Steps to reproduce:
+endif::[]
 
-. Create a guided scorecard with a name that begins with a lowercase letter.
-. Import the guided scorecard into {CENTRAL}.
-. Click *Test*.
 
-Expected result: The test completes successfully.
+== {CENTRAL}
 
-Actual result: The test fails.
 
-Workaround: Make sure that guided scorecard names do not begin with lowercase letters.
 
-.You cannot add a new group in {CENTRAL} integrated with RH-SSO [https://issues.jboss.org/browse/RHPAM-2036[RHPAM-2036]]
+.An error might occur if you import a sample project twice [https://issues.jboss.org/browse/RHPAM-2434[RHPAM-2434]]
 
-Issue: The following exception occurs when you try to add a new group in {CENTRAL} if {CENTRAL} is integrated with RH-SSO 7.3.0:
-[source]
-----
-java.lang.NoSuchMethodError:org.keycloak.representations.idm.RoleRepresentation.setScopeParamRequired(Ljava/lang/Boolean;)
-----
+Issue: An `Unable to complete your request` error might appear if you import a sample project that has already been imported.
 
 Steps to reproduce:
 
-. Open {CENTRAL}.
-. Open the *Admin* page.
-. Open *Groups*.
-. Try to create a new group.
+* Import a sample project twice. Note that this error does not always occur.
 
+Workaround: Remove the originally imported sample project then import the sample project again.
```

Comment:
I have issue with 'originally imported'. I am afraid that would tell users to go to someone else's project and remove it.
I'd provide more info, sth like:
If the project whose import failed is present in your space, remove it first. Then import it again.

Reply:
OK, I rewrote. Please check.

Reply:
Thanks, looks good.